NOLF2/ClientShellDLL/TO2/GlobalsInit.cpp

```diff
@@ -19,5 +19,18 @@ define_holder(ILTTexInterface,g_pTexInterface);
 
 SETUP_CLIENTSHELL();
 
+#ifndef __LINUX
 define_interface(CTO2GameClientShell, IClientShell);
-
+#else
+static CAPIInstanceDefines<CTO2GameClientShell>
+	__impl_CTO2GameClientShell_defines_IClientShell_Default__(
```

Comment:
as per https://en.cppreference.com/w/cpp/language/identifiers
> the identifiers with a double underscore anywhere are reserved;
the identifiers that begin with an underscore followed by an uppercase letter are reserved;
the identifiers that begin with an underscore are reserved in the global namespace.

so to avoid possible UB I'll be cleaning up names and defines with double underscores __